### PR TITLE
including additional files .deb package

### PR DESCRIPTION
**Pull request by yeban**
@tbooth

So the approach I took with including additional files (`context/`) in .deb package is like this:
1. move `context/`  to `share/oswitch/` within repository.
2. introduce a constant called `PREFIX` in `pkg.rb` initialised to path of repo root.
3. read contents of `context/` as `PREFIX/share/oswitch/context/`.
4. replace our `pkg.rb` with a Debian specific one (`debian/pkg.rb`) (using `debian/install` file) that overrides `PREFIX` to `/usr`.

What do you think?

### CODE REVIEW

File: debian/install [hunk 1]
```diff
@@ -0,0 +1,2 @@
+share usr
+debian/pkg.rb usr/lib/ruby/vendor_ruby/oswitch
```
Comment (yeban) [inline debian/install]: @tbooth: Does `usr/lib/ruby/vendor_ruby/oswitch` have to be hard-coded here? Is there some magic variable that I can use instead?

**Comment by tbooth** [comment]
Hi,
I've looked at the package and made some small modifications.  Having the identical debian/control file for Trusty and Jessie is a little ugly but can be made to work - see fixups in debian/rules.

re. Magic variables in debian/install.  Nope, and I don't see why you really want one.  Too many magic variables obfuscates debugging.  Remember that from the packagers perspective these are well-known fixed paths, not things to be configured.

Your pkg.rb fix looks fine.  Generally where I see relative paths in something i want to package I have to patch them using quilt, which adds another maintenance task.

I've tested build/install for this and it looks good to me.  Not actually tested running a container yet though.

Cheers,
TIM

**Comment by yeban** [comment]
Thanks for reviewing. I will move this to master. Regarding wanting magic variables, I wasn't thinking that `usr/lib/ruby/vendor_ruby/oswitch` would be a well known path.

**Comment by yeban** [comment]
rebased and merged

**Pull request closed without merging**